website/docs/contributing.md

```diff
@@ -25,42 +25,46 @@ When opening an issue, please select the most appropriate template for what you'
 
-1. [Docker](https://docs.docker.com/get-docker/)
-   > For Windows users, enabling [WSL2 back-end integration with Docker](https://docs.docker.com/docker-for-windows/wsl/) is recommended.
-2. [Visual Studio Code](https://code.visualstudio.com/)
-3. [Visual Studio Code Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+### Docker Setup
 
-> **⚠ If running via Docker Desktop for Windows**
->
-> Note that the [mounted workspace files appear owned by `root`](https://code.visualstudio.com/remote/advancedcontainers/add-nonroot-user) in the dev container, which will cause `git` commands to fail with a `fatal: detected dubious ownership in a repository` error due to [safe.directory](https://git-scm.com/docs/git-config/2.35.2#Documentation/git-config.txt-safedirectory) checks. This can be addressed by changing the mapped ownership of the workspace files in the dev container to the `vscode` user:
->
-> ```bash
-> sudo chown -R vscode:vscode /workspace/copacetic
-> ```
+Copacetic requires Docker for patching some images. To install Docker, follow the [Docker installation guide](https://docs.docker.com/engine/install/).
+
+If running Copa gives you the following error:
+
+```bash
+Error: could not use docker driver: failed to solve: requested experimental feature mergeop  has been disabled on the build server: only enabled with containerd image store backend
+requested experimental feature diffop  has been disabled on the build server: only enabled with containerd image store backend
+missing required buildkit functionality
+could not use buildx driver: listing workers for Build: failed to list workers: Unavailable: connection error: desc = "transport: Error while dialing: open /home/gitpod/.docker/buildx/current: no such file or directory"
+could not use buildkitd driver: %!w(<nil>)
+```
 
-#### Personalizing user settings in a dev container
+You will need to [enable containerd image store support](https://docs.docker.com/storage/containerd/) within Docker Engine.
 
-VSCode supports applying your user settings, such as your `.gitconfig`, to a dev container through the use of [dotfiles repositories](https://code.visualstudio.com/docs/remote/containers#_personalizing-with-dotfile-repositories). This can be done through your own VSCode `settings.json` file without changing the dev container image or configuration.
+> Note: Docker can eat up your disk space. If you find that your storage is rapidly being taken up after working with Copa, run `docker system prune`. This will prune all unused images, containers and caches. Running this command will not break anything. This is an issue with Docker, not with Copa.
```

Comment:
Could we add this to the FAQ section instead?

Reply:
Regarding the error from lines 47-55, it looks like this is already mentioned in the quick start docs. I've removed the error from my PR so we don't repeat the same information.

Reply:
can we also remove it from the CONTRIBUTING.md file to keep it consistent?